ipatests/prci_definitions/gating.yaml

```diff
@@ -298,3 +298,15 @@ jobs:
         template: *ci-ipa-4-9-latest
         timeout: 3600
         topology: *master_1repl
+
+  fedora-latest/test_subids:
```

Comment:
To follow the implicit naming convention, the task should be fedora-latest-ipa-4-9, and it requires [fedora-latest-ipa-4-9/build], with build_url '{fedora-latest-ipa-4-9/build_url}' + template *ci-ipa-4-9-latest

Reply:
Fixed, thanks Flo!